themes/10up-theme/TAILWIND.md

```diff
@@ -44,8 +44,7 @@ Tailwind allows you to author custom classes that group multiple utilities toget
 		border-blue-800
 		hover:bg-blue-800
 		hover:text-white
-		rounded-lg
-		bg-white;
```

Comment:
Why do we need to remove the background here?

Reply:
@iansvo it was repeated in the lines above :)